### PR TITLE
Add roles documentation

### PR DESCRIPTION
**Pull request by pdcp1**
This is part of https://github.com/responsible-ai-collaborative/aiid/issues/2383

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -377,6 +375,24 @@ Add the following environment variable on Netlify:
...
+
+As soon as a user is signed in, the system assign a `subscriber` role as default. Role assignment is handled manually by the site administrators.
+
+**The roles are:**
+
+| User Role                     | Permissions                                                                                                                             |
+|-------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|
+| `subscriber`                  | This is the default role assigned to all users. It allows the user to subscribe to new incidents, specific incidents, entities, and anything else that is subscribeable. |
+| `submitter`                   | This role allows the user to submit new incidents under their user account.                                                            |
+| `incident_editor`             | This role allows the user to:<br>- Edit and clone incidents<br>- See the live incident data<br>- Add, edit, approve and delete Variants<br>- View and submit incident candidates<br>- Restore previous versions of incidents and reports. |
```
Comment (cesarvarela) [inline README.md:390]: Maybe add a quick explanation of what we mean with "live incident data"?

Comment (pdcp1) [reply README.md]: Added

File: README.md [hunk 2]
```diff
@@ -377,6 +375,24 @@ Add the following environment variable on Netlify:
 12. In `Triggers` select `A document is published` and `A document is unpublished`
 13. Click `Add this webhook`
 
+## User Roles
+
+All site users have one or more roles assigned to them. The role determines what actions the user can take on the site.
+
+As soon as a user is signed in, the system assign a `subscriber` role as default. Role assignment is handled manually by the site administrators.
```
Comment (kepae) [inline README.md:382]:
```suggestion
As soon as a user is signed in, the system assigns a `subscriber` role by default. Role assignment is handled manually by the site administrators.
```

File: README.md [hunk 3]
```diff
@@ -377,6 +375,24 @@ Add the following environment variable on Netlify:
...
+|-------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|
+| `subscriber`                  | This is the default role assigned to all users. It allows the user to subscribe to new incidents, specific incidents, entities, and anything else that is subscribeable. |
+| `submitter`                   | This role allows the user to submit new incidents under their user account.                                                            |
+| `incident_editor`             | This role allows the user to:<br>- Edit and clone incidents<br>- See the live incident data. The live data is the data that is currently stored in the database. Keep in mind that the Incident page is generated on each build, so if a user edits an incident, the change will be only visible if the live data options is activated until the next build finishes.<br>- Add, edit, approve and delete Variants<br>- View and submit incident candidates<br>- Restore previous versions of incidents and reports. |
```
Comment (kepae) [inline README.md:390]:
```suggestion
| `incident_editor`             | This role allows the user to:<br>- Edit and clone incidents<br>- See the live incident data. The live data is the data that is currently stored in the database. Keep in mind that the incident page is generated on each build, so if a user edits an incident, the change will be only visible if the live data options is activated until the next build finishes.<br>- Add, edit, approve and delete incident variants<br>- View and submit incident candidates<br>- Restore previous versions of incidents and reports. |
```

**Review by kepae — APPROVED** [review]

**Review by kepae — COMMENTED** [review]
Nice! Minor grammar/style suggestions.

File: README.md (hunk 2 above)
Comment (pdcp1) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (pdcp1) [reply README.md]: Done

**Comment by pdcp1** [comment]
@kepae The changes requested are done. Please review

**Pull request merged**